cmd/commands/repo.go

```diff
@@ -356,36 +358,54 @@ func NewRepoUninstallCommand() *cobra.Command {
 
 	<BIN> repo uninstall --repo https://github.com/example/repo --force
 `),
-		PreRun: func(_ *cobra.Command, _ []string) { cloneOpts.Parse() },
+		PreRun: func(_ *cobra.Command, _ []string) {
+			if !clusterOnly {
+				cloneOpts.Parse()
+			}
+		},
 		RunE: func(cmd *cobra.Command, _ []string) error {
 			kubeContextName, err := cmd.Flags().GetString("context")
 			if err != nil {
 				return fmt.Errorf("failed to get kube context name: %w", err)
 			}
+
+			if !clusterOnly {
+				if cloneOpts.Repo == "" || cloneOpts.Auth.Password == "" {
```

Comment:
check if cobra can handle this

Reply:
i would like to define both "repo" and "token" as required if "clusterOnly" is false. i don't think cobra has anything specific for that, other then doing the check in code. i moved it to PreRunE, but basically left it the same.